utility/src/main/java/org/oppia/android/util/threading/BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+"""
+General purpose utilities for Dispatcher module and Concurrent collections.
```

Comment:
Perhaps instead: ``General purpose utilities for multi-threading and concurrency.``

This might be a nice package to move more of these types of utilities to in the future.

Reply:
Done.